core/domain/rights_manager.py

```diff
@@ -180,22 +208,50 @@ def _save_activity_rights(
     model.commit(committer_id, commit_message, commit_cmds)
 
 
-# Update summary of changed activity (note that the activity rights id is the
-# same as the activity id).
-# TODO(msl): get rid of inline imports by refactoring code
+# TODO(msl): get rid of inline imports by refactoring code.
```

Comment:
Could we put this just above the offending line (`from core.domain import exp_services`)?